doc/release-notes.md

```diff
@@ -274,6 +277,12 @@ To enable block pruning set `prune=<N>` on the command line or in
 `bitcoin.conf`, where `N` is the number of MiB to allot for
 raw block & undo data.
 
+A value of 0 disables pruning. The minimal value above 0 is 550. Your
+wallet is as secure with high values as it is with low ones. Higher
+values merely reduce the network traffic in case of reorganization of
```

Comment:
Thanks for your explanations. Fixed by 54d390780feaef895fede8da3f436b932af29c1d

With regards to the multiple options of resolving this, here's what I decided:

> Rather than try to re-explain pruning here, maybe it would make sense to refer readers back to the 0.11.0 release notes? I don't know how easy that is to link to I guess.

I think it makes sense to explain it here for the usability purpose of having a self-contained document. However, linking the old release notes also is useful. So I just did both.

> I agree this misunderstands things a bit. The minimum value is chosen to handle any realistic reorganization. Larger reorgs will be disastrous for many other reasons, so there's no need to even consider them here.

If the node can crash, that is worth mentioning at least? For example, it could cause financial loss for miners if their nodes shut down.
To find a tradeoff between your suggestion of not mentioning it at all, and the "crash + reindex required" information, I tried to keep it as neutral as possible by saying the node will "shutdown" (spelling of that word fixed in 15c0263ff1e4f4773005291e1eb584954900e197), and leaving out what has to be done afterwards. That both conveys the information that it won't work anymore, and allows leaving out what happens afterwards by not scaring the user off with a strong word such as "crash".

> If you want to link to older release notes the best way, I think, would be to link to the historical release notes in master: https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.11.0.md#block-file-pruning .
Alternatively you can link to the current release notes at the time of the v0.11.0 tag:
https://github.com/bitcoin/bitcoin/blob/v0.11.0/doc/release-notes.md#block-file-pruning
These are probably the same but may potentially be missing some post-facto editorial fixes.

With regards to technical documentation, I think non-permanent links are an annoyance: They make it difficult to reconstruct what a user did read, since the document he saw will vary depending on when he accessed the link.
So I chose to link the tag.
Luckily, the two files are binary equivalent currently anyway.